src/views/Assignment/js/assignment-detail.js

```diff
@@ -128,37 +92,6 @@ export default {
           assignmentId: this.assignmentDetail.id
```

Comment:
this might not be used anymore as the room list call `StudentListWithinBatch` API that don't require `assignmentId`

Reply:
cc: @DavidWilliamK